woocommerce/woocommerce-admin
# Make pagination buttons height and width consistent

Fixes #6709 

Make pagination buttons height and width consistent

### Screenshots

https://user-images.githubusercontent.com/56378160/113228312-92117500-9262-11eb-9ea7-fc7a50b89d58.mp4

### Detailed test instructions:
1. Go to Marketing > Overview
1. Scroll down to the WooCommerce knowledge base section
1. Click the pagination buttons and see the width and height are consistent
